src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/DebuggerTraceListener.cs

```diff
@@ -38,17 +38,17 @@ public Task InitializeAsync(IAsyncServiceProvider asyncServiceProvider)
 
         public override void Write(string message)
         {
-            if (Debugger.IsLogging())
+            if (System.Diagnostics.Debugger.IsLogging())
```

Comment:
A package upgrade must have introduced a `Debugger` class into a `Microsoft.*` namespace, as this broke. A using alias didn't help either, hence the fully qualified name here.

Reply:
If you put `using System.Diagnostics;` inside the namespace (not above it like it is now), it would have used the right debugger class as an alternative to fully qualifying it.